packages/microsoft-reactnative-sampleapps/windows/SampleLibraryCS/SampleLibraryCS.csproj

```diff
@@ -113,7 +114,6 @@
     <Compile Include="CustomUserControlCS.cs" />
     <Compile Include="CustomUserControlViewManagerCS.cs" />
     <Compile Include="CircleViewManagerCS.cs" />
-    <Compile Include="ReactPackageProvider.cs" />
```

Comment:
I think this is colliding with @jonthysell 's autolinking stuff... can you guys sync up? :)

Reply:
Where / when does the ReactPackageProvider get generated?

Reply:
During the build in the obj directory. I checked with your autolink changes for managed code and they run separate tracks. My codegne generates the types for the current assembly, Jon's logic imports and adds registrations for dependencies. The only place where there would be overlap is in the 'production' side, but I haven't found that code yet.